src/triage/component/results_schema/nuke_triage.sql

```diff
@@ -0,0 +1,48 @@
+/*
+  Function for wiping all triage tables, schemas and indexes.
+Useful  when ou try to start clean (again) and try to recover some
+previous errors
+ */
+create or replace function nuke_triage()
```

Comment:
The presence of a `nuke_triage` function can have some utility, but certainly also makes me nervous about the potential for accidental use. Could we maybe consider some way of making the user confirm they really mean it? (one hacky way I've done this in SQL before is to check for the presence of a certain value in a certain table, for instance, maybe they need to create `model_metadata.nuke_triage` with a value of `True` in a `confirm_nuke` column or something?)

Reply:
You know, with a great power a great responsability ...

But it is like being afraid of `drop database` ...

Reply:
Yeah, that's a fair point, I suppose `drop database` doesn't really ask for a confirmation either, though I'm also not entirely sure that means it's good design or behavior I'd want to emulate 😄 

Anyway, I don't think it's a critical thing to hold up the PR, but might be worth considering at some point...